### ISSUE TITLE
[BUG]Unicode characters in button text

### ISSUE THREAD

**Issue by isaacclausen** [bitfocus/companion#2731]
### Is this a bug in companion itself or a module?

- [x] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

In 3.2.0 on a Ventura iMac, unicode characters seem to mess up button text. When a unicode character is in the string, the line spacing is almost doubled.

Buttons with unicode characters in 3.1.2:
<img width="222" alt="Screenshot 2024-02-02 at 9 53 09 AM" src="https://github.com/bitfocus/companion/assets/83613660/2de5d6a0-52a7-43d8-b258-9f920b137c15">

Buttons with unicode characters in 3.2.0:
<img width="392" alt="Screenshot 2024-02-02 at 9 50 36 AM" src="https://github.com/bitfocus/companion/assets/83613660/3bdbd9b3-f67e-42ce-8d56-cf33ced021d1">
<img width="528" alt="Screenshot 2024-02-02 at 9 51 35 AM" src="https://github.com/bitfocus/companion/assets/83613660/5089c136-2ecc-4bd8-a682-a562146b5a1d">

### Steps To Reproduce

1. Add unicode character to button text

### Expected Behavior

Unicode characters display properly without added line spacing

### Environment (please complete the following information)

```markdown
- OS:Ventura
- Browser:Safari 16.4
- Companion Version:3.2.0
```

**Comment by jswalden** [comment]
FWIW on Sonoma with 3.2.0 and...some version of Safari, not sure which one (the computer is not immediately on hand for me to check), at least _some_ emojis display for me:

<img width="878" alt="unicode" src="https://github.com/bitfocus/companion/assets/4139753/dcdd1785-890a-4b2f-b094-858ae51eef09">

(It's possible I should be quibbling about the baseline on that bomb emoji -- or about the overflow behavior on the button display, versus the buttons displaying smaller -- but those would be separate issues.)

**Comment by isaacclausen** [comment]
Here a video showing the problem better, when I add a unicode character, it adds extra unnecessary line spacing:
https://github.com/bitfocus/companion/assets/83613660/0854905f-ff2f-4b46-9c89-cb6a27834a84

**Comment by dnmeid** [comment]
Hi @isaacclausen 
I'm aware of this problem. Actually it is not a problem of unicode, but of the linespacing in the fonts we are using. The font we are using for the latin characters has a different spacing from the font we are using for the symbols.
I have a solution with adjusted fonts here but didn't have the time to upload and test it.

**Comment by Julusian** [comment]
This is resolved in the latest release (v3.4.0)

**Comment by dnmeid** [comment]
Interesting, that seems to be a different problem, not related to the original line-spacing stuff.
